sdcm/cluster.py

```diff
@@ -974,7 +974,8 @@ def get_backtraces(self):
             if self.is_ubuntu14():
                 # fixme: ubuntu14 doesn't has coredumpctl, skip it.
                 return
-            core_pids = self._get_core_pids()
+            core_pids = None
+            # core_pids = self._get_core_pids()
```

Comment:
Why this line commented?

Reply:
oh, it got in this PR by mistake...
i commented it out because of #2217, but it shouldn't get in the code...